sdk/eventhub/event-hubs/src/eventHubProducerClient.ts

```diff
@@ -139,6 +139,16 @@ export class EventHubProducerClient {
     this._producersMap = new Map();
   }
 
+  /**
+   * Creates an instance of `EventDataBatch` to which one can add events until the maximum supported size is reached.
+   * The batch can be passed to the `send()` method of the `EventHubProducer` to be sent to Azure Event Hubs.
+   * @param options  A set of options to configure the behavior of the batch.
+   * - `partitionKey`  : A value that is hashed to produce a partition assignment.
+   * Not applicable if the `EventHubProducer` was created using a `partitionId`.
+   * - `maxSizeInBytes`: The upper limit for the size of batch. The `tryAdd` function will return `false` after this limit is reached.
+   * - `abortSignal`   : A signal the request to cancel the send operation.
```

Comment:
```suggestion
   * - `abortSignal`   : A signal the request to cancel the `createBatch` operation.
```

Reply:
Fixed with https://github.com/Azure/azure-sdk-for-js/pull/5964/commits/92eb64d98eb395114ec3f7f761db85106344c2f8